src/app/components/ProductGrid.tsx

```
@@ -1,15 +1,234 @@
+                  {product.currency || '$'}{product.price}
+                </span>
+              </div>
+
+              {/* Rating & Reviews */}
+              {product.rating && (
+                <div className="flex items-center gap-2 mb-3">
+                  <div className="flex items-center">
+                    {[...Array(5)].map((_, i) => (
+                      <svg
+                       aria-hidden="true"
+                        key={i}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)